docs/connect-a-frontend.md

```diff
@@ -0,0 +1,20 @@
+This demo will discuss the options we would have if we wanted to connect a frontend to the AEA. The following diagram illustrates 
+the two options we are going to discuss.
+
+<center>![How to connect frontend to your AEA](assets/http-integration.png)</center> 
+
+## Case 1
+The first option we have is to create a connection that will handle the incoming requests from the rest API. In this scenario,
+the rest API communicates with the AEA and requests are handled from a connection that we developed. The rest API should send envelops to the AEA, then the multiplexer will pass the message to the correct skill. 
+
+## Case 2
+The other option would be to create a stand-alone `Multiplexer` with an `OEF` connection that will be able to search and discover other AEAs. In this scenario,
```

Comment:
with an `OEF` connection. In this scenario, the frontend needs to incorporate a Multiplexer with an OEF connection. Then the OEF messaging relay can be used to send envelopes from the AEA to the frontend.